composer.json

```diff
@@ -10,8 +10,10 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
-        "psr/http-message": "^1.0"
+        "php": "^5.3 || ^7.0"
+    },
+    "suggest": {
+        "psr/http-message": "The package containing the PSR-7 interfaces"
```

Comment:
Since this package was intended as a direct utility for the PSR-7 package it makes sense to suggest it.
But in hinsight of what the package currently provides (generally usable HTTP related constants), it isn't even necessary to suggest the other package.

Reply:
I think the suggestion is still a good idea; it's not saying you MUST install them, but it IS indicating that the primary reason they exist is for that package.